apps/challenges/views.py

```diff
@@ -936,6 +936,9 @@ def create_challenge_using_zip_file(request, challenge_host_team_pk):
 
             zip_config.challenge = challenge
             zip_config.save()
+
+            send_slack_notification(
+                message="A *new challenge* has been uploaded to EvalAI")
```

Comment:
This message is not informative. Please stringify a dict that contains 'link of the zip uploaded', 'primary key of uploaded config' and 'Challenge name' (if a challenge is created).